README.md

```diff
@@ -48,17 +48,24 @@ Step 1: Clone the repository
 Step 2: Download the datasets and move the data files to the dataset directory
 
   ```sh
-  wget -r -np -R "index.html*"  http://43.138.115.238/download/febench/data/; cp -r <dataset directory> ./dataset
+  wget -r -np -R "index.html*"  -nH --cut-dirs=3  http://43.138.115.238/download/febench/data/  -P ./dataset
   ```
 
 > Note the data files are in parquet format.
 
-Step 3: Start the cluster and enter the *OpenMLDB* folder
+Step 3: [Start the cluster](https://openmldb.ai/docs/zh/main/deploy/install_deploy.html) and enter the `./OpenMLDB` folder in this project
+> Please be aware that the default values for `spark.driver.memory` and `spark.executor.memory` may not be enough for your needs. If you encounter a `java.lang.OutOfMemoryError: Java heap space` error, you may need to increase them. You can refer to [this document](https://openmldb.ai/docs/zh/main/maintain/faq.html#java-lang-outofmemoryerror-java-heap-space) for guidance. One acceptable size for these parameters is 32G/32G.
+
+Step 4: Please modify the `conf.properties.template` file to create your own `conf.properties` file in the `./conf` directory, and update the configuration settings in the file accordingly, including the OpenMLDB cluster and the locations of data and queries. 
+
+```sh
+cp ./conf/conf.properties.template ./conf/conf.properties
+vim ./conf/conf.properties
```

Comment:
vim and change the path every line? Inefficient. Don't use the long name `<absolute path to project>`, just `<path>`. And 
```
export DATASET_ROOT=`pwd`/dataset
sed s#<path>#$DATASET_ROOT# ./conf/conf.properties.template > ./conf/conf.properties
```